include/deal.II/lac/petsc_precondition.h

```diff
@@ -562,11 +562,12 @@ namespace PETScWrappers
 
 
   /**
-   * A class that implements the interface to use the PETSc LU preconditioner (@p PCLU).
+   * A class that implements the interface to use the PETSc LU preconditioner
+   * (@p PCLU). This usually performs an exact factorization of the matrix, so
```

Comment:
The word 'usually' in here is ambiguous. Is this controlled by an option?

Reply:
Yes. There is a bunch of options in there that might not give you an exact factorization (pivoting, damping).